libs/message-coder/vitest.config.mts

```diff
@@ -0,0 +1,5 @@
+import { defineConfig } from '../../vitest.config.shared.mjs';
+
+export default defineConfig({
+  test: {},
+});
```

Comment:
Is there a point to passing in `{ test: {} }` here or could there just be no parameter?